src/main/java/calculator/Numbers.java

```diff
@@ -11,18 +12,14 @@ public Numbers(List<Number> numbers) {
     }
 
     public static Numbers create(String[] tokens) {
-        List<Number> numbers = new ArrayList<>();
-        for (String token : tokens) {
-            numbers.add(new Number(token));
-        }
-        return new Numbers(numbers);
+        return new Numbers(Arrays.stream(tokens).map(Number::parse).collect(Collectors.toList()));
```

Comment:
스트림은 대부분 위에서 아래로 읽히기 때문에, 한줄에 점 하나의 규칙을 따라 작성하긴 합니다.
```suggestion
    return new Numbers(Arrays.stream(tokens)
                .map(Number::parse)
                .collect(Collectors.toList()));
```

이런식으로요 :)

Reply:
한줄에 하나로 작성하니 훨씬 보기 쉬워지네요!
수정 했습니다~!

https://github.com/next-step/java-racingcar/pull/3786/commits/6b15e2b127c835b50ec5389f6d54b7d6523403ea